e2e/testdata/fn-render/empty-pipeline/.expected/config.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google LLC
```

Comment:
nit 1: how come some files have headers not others.
nit 2: exitCode of zero is default and can be ommited (probably the entire file as well)

Reply:
Good catch on `1`. I did not run the make target to add license file, just did that, now all files have the license header. Presubmit should catch this probably. Will file an issue for this.

Removed the `config.yaml` where expected config matches the defaults.